root/tree/dataframe/RDataFrameOpenDataBenchmarks.cxx

```diff
@@ -552,7 +554,7 @@ void benchmark8_jitted(unsigned int nthreads)
               .Define("Lepton_flavour", "Concatenate(ROOT::RVec<int>(nMuon, 0), ROOT::RVec<int>(nElectron, 1))")
               .Define("AdditionalLepton_idx", benchmark8_additional_lepton_idx,
                       {"Lepton_pt", "Lepton_eta", "Lepton_phi", "Lepton_mass", "Lepton_charge", "Lepton_flavour"})
-              .Filter("AdditionalLepton_idx != -999", "No valid lepton pair found.")
+              .Filter("AdditionalLepton_idx != PLACEHOLDER_VALUE", "No valid lepton pair found.")
```

Comment:
I don't think this can work...? `PLACEHOLDER_VALUE` is a compiled entity, the ROOT interpreter won't know about it